config/build/dpkg_build/postinst.sh

```diff
@@ -72,8 +72,10 @@ else
    # update tutorial
    #
    ###########################################################################
-   rm -r ${HOME}/RobotTest/tutorial/*
-   cp -R -a /opt/rfwaio/robotvscode/RobotTest/tutorial/. ${HOME}/RobotTest/tutorial
+   if [ -d ${HOME}/RobotTest/tutorial ]; then
+      rm -rf ${HOME}/RobotTest/tutorial/*
+   fi
+   cp -R -a /opt/bosch/robfw/robotvscode/RobotTest/tutorial/. ${HOME}/RobotTest/tutorial
```

Comment:
Hi Nguyen,
you accidentally mix paths.
Sometimes you use ```/opt/bosch/robfw```,
sometimes ```/opt/rfwaio```

```/opt/rfwaio``` is correct.
Thank you,
 Thomas